MovieAppPetProject/MovieAppPetProject/GetMovieDetailsByID/GetMovieDetails-ViewController.swift

```diff
@@ -8,10 +8,35 @@
 import UIKit
 
 class MovieDetailsViewController: UIViewController {
-    private let movieDetailsViewModel = MovieDetailsViewModel()
-
+    private var selectedSearchMovie: SearchMoviesResults?
+    private var selectedHomeScreenMovies: NowPlayingResults?
+    
+    @IBOutlet weak var titleLabel: UILabel!
+    @IBOutlet weak var posterImageView: UIImageView!
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-        movieDetailsViewModel.fetchMovieDetails()
+        displayMovieDetails()
+    }
+    
+    func setMovieDetails(selectedSearchMovie: SearchMoviesResults? = nil, selectedHomeScreenMovies: NowPlayingResults? = nil) {
```

Comment:
can omit = nil if it is optional. swiftLint should have picked that up😄

Reply:
rather take the results and store them in the viewModel. Have a look at Gia's PR for reference.😄
So you receive the stuff on this screen, bur then have a viewModel for this screen and pass it to the view Modelcan be private😄